commit 46010b9d50aed454e9c9e5347df10e6cbb6eac39
Author: Ake Sandgren <ake.sandgren@hpc2n.umu.se>
Date:   Mon Mar 10 11:13:33 2025 +0100

    Skip test_var_mean_differentiable
    It sometimoes gets a segfault during garbage collection causing problems in PASS/FAIL reporting

diff --git a/test/test_autograd.py b/test/test_autograd.py
index 791510e4cc4..94a826c11ec 100644
--- a/test/test_autograd.py
+++ b/test/test_autograd.py
@@ -3881,6 +3881,7 @@ class TestAutograd(TestCase):
         with self.assertRaisesRegex(Exception, 'Simulate error'):
             d.sum().backward()
 
+    @unittest.skip("Occasionally gets a segfault during garbage collection")
     def test_var_mean_differentiable(self):
         dim = [2, 4]
         keepdim = False
